velodyne_pointcloud/tests/CMakeLists.txt

```diff
@@ -10,10 +10,18 @@ ament_target_dependencies(test_row_step
   velodyne_msgs
 )
 
-# C++ gtests
-# catkin_add_gtest(test_calibration test_calibration.cpp)
-# add_dependencies(test_calibration ${catkin_EXPORTED_TARGETS})
-# target_link_libraries(test_calibration velodyne_rawdata ${catkin_LIBRARIES})
+ament_add_gtest(test_calibration
+  test_calibration.cpp
+  ../src/lib/calibration.cpp
```

Comment:
Relative file locations should not be used. If you require a file outside of this folder, please move these tests into the root CMakeLists.txt file. This is the common setup anyway.

Reply:
addressed at https://github.com/ros-drivers/velodyne/pull/452/commits/b420863c74a320e1c19e7e611c013151d08f022d